docs/analytics/resources/emr-vars.yml

```diff
@@ -5,7 +5,7 @@ region: 'us-east-1'
 instance_groups:
   master:
     num_instances: 1
-    type: m4.medium
+    type: m4.large
```

Comment:
Not totally related to this doc PR - but could you remind me why we needed to size up? Was it to speed up ProblemResponseReportWorkflow for epodx?